packages/manager/modules/overthebox/src/overthebox/autoconfigure/dns/autoconfigure-dns.controller.js

```diff
@@ -0,0 +1,240 @@
+export default class OverTheBoxAutoconfigureDns {
```

Comment:
We have as habit to suffix with `controller` the controller classes.

```suggestion
export default class OverTheBoxAutoconfigureDnsController {
```